packages/client/tests/functional/interactive-transactions/tests.ts

```diff
@@ -401,7 +401,7 @@ testMatrix.setupTestSuite(({ provider }) => {
   /**
    * Makes sure that the engine does not deadlock
    */
-  test('high concurrency', async () => {
+  testIf(provider !== 'sqlite')('high concurrency', async () => {
```

Comment:
Add a comment why we need to skip?

Reply:
This is flaky because of the file-based nature of sqlite. The test was previously skipped but got removed during test porting.